src/components/Comments/Comment.js

```diff
@@ -179,6 +252,13 @@ const Comment = ({
             </ButtonHightlight>
           </TouchableOpacity>
         )}
+        <ButtonHightlight
+          onLongPress={handleOnLongPress}
+          style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+          onPress={() => refSheet.current.open()}>
+          <MemoSendDM width={20} height={18} />
+        </ButtonHightlight>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a new button for sending direct messages is a good feature. However, it's important to ensure that this button is accessible and provides adequate feedback to users, especially when an action is being processed or has failed.

```diff
+ <ButtonHightlight
+   onLongPress={handleOnLongPress}
+   style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+   onPress={() => refSheet.current.open()}>
+   <MemoSendDM width={20} height={18} />
+ </ButtonHightlight>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->